### PR TITLE
Update to Rails 3.2.21

### PR DESCRIPTION
**Pull request by mattr-**
Update Rails to 3.2.21 because 3.2.12 is old.

### CODE REVIEW

**Comment by AaronLasseigne** [comment]
The README specifically mentions the Rails version. We should update that too or remove it.

**Comment by mattr-** [comment]
Good call. :+1: I updated it.

File: Gemfile.lock [hunk 1]
```diff
@@ -205,20 +204,209 @@ GEM
...
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
+      polyglot (>= 0.3.1)
       polyglot (>= 0.3.1)
```
Comment (ckrailo) [inline Gemfile.lock]: uhhhhhh

Comment (ckrailo) [inline Gemfile.lock]: @mattr- any idea how that happened?

**Comment by ckrailo** [comment]
We should probably hop to 4.2 at some point, but this is the first step. :)

**Comment by mattr-** [comment]
I just did a `bundle update`. No idea. Aren't you the bundler guy in the group?

 :wink:

**Comment by AaronLasseigne** [comment]
I just tried this and got the exact same output. :frowning:

**Comment by mattr-** [comment]
ohai! i finally fixed it.

**Pull request merged**